lib/utils.js

```diff
@@ -96,6 +96,9 @@ function patchEventTargetMethods(obj) {
   // This is required for the addEventListener hook on the root zone.
   obj[keys.common.addEventListener] = obj.addEventListener;
   obj.addEventListener = function (eventName, handler, useCapturing) {
+    if (!handler || (typeof(handler) !== 'function' && typeof(handler.handleEvent) !== 'function')) {
+      return;
```

Comment:
ah...It's my fault. I was wrong to attach https://github.com/angular/zone.js/issues/167 to eventListener's issue.  It seem to be another issue.

Anyway, when you fire `addEventListener('someEvent', {a: 123})` or `addEventListener('someEvent', {handleEvent: 'something not a function'})` in browser without zone.js, nothing would happen and have no effect. But when you are firing these in with zone.js, It would throw error. I think it doesn't seem like a good behavior. 

And about the issue about IE9, I had considered about it. Then codes would return when the handler isn't function and handler.handleEvent isn't function. It has no effect to the previous codes.